groups/data-lake/main.tf

```diff
@@ -14,6 +10,24 @@ data "aws_security_group" "mongo_db" {
   }
 }
 
+data "vault_generic_secret" "secrets" {
+  path = "applications/${var.aws_profile}/data-lake"
+}
+
+# ------------------------------------------------------------------------------
+# Locals
+# ------------------------------------------------------------------------------
+locals {
+  database_password = data.vault_generic_secret.secrets.data.database_password
+  database_username = data.vault_generic_secret.secrets.data.database_username
+  mongo_export_db_url = data.vault_generic_secret.secrets.data.mongo_export_db_url
+  mongo_export_s3_path = data.vault_generic_secret.secrets.data.mongo_export_s3_path
```

Comment:
Can we align the `=` character in each variable declaration here so make the code more consistent with other parts of the code?